Doc/library/http.server.rst

```diff
@@ -520,6 +520,12 @@ the ``--cgi`` option::
    :mod:`http.server` command line ``--cgi`` support is being removed
    because :class:`CGIHTTPRequestHandler` is being removed.
 
+.. warning::
+
+   :class:`CGIHTTPRequestHandler` and the ``--cgi`` command line option
+   have known security issues, therefore you should not expose them
+   to untrusted clients. Use it at your own risk.
```

Comment:
```suggestion
   are not intended for use by untrusted clients and may be vulnerable
   to exploitation. Always use within a secure environment.
```

How about this? I think it's a little less down on ourselves, and hopefully gives users something they can actually do (i.e. "I should secure my environment")